### PR TITLE
docs: update README with a content optimizations for SEO

### PR DESCRIPTION
**Pull request by ala-n**
Use this links to see changes live:
 - https://github.com/exadel-inc/esl/tree/docs/readme-content-upd
 - https://github.com/exadel-inc/esl/blob/docs/readme-content-upd/README.md
 
 Original docs
 - https://docs.google.com/document/d/1AJVIekEZ0BrhJA8gDiUjf9rBqgzDnjuwJMzyUdJATCU/edit

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,22 +9,46 @@
...
+and lets you efficiently create websites with multiple web components at hand. 
+While looking for the best UI component library, keep in mind that it should be compatible with other libraries 
+and frameworks. And ESL is!
+
+The UI component library allows you to access the toolkit full of reusable components. 
+You can utilize these web components to complement projects or build your infrastructure from the ground 
+up using ESL alone. Our web UI components library is accurate and you can easily work around styles 
+to make your projects consistent.
+
+ESL based just on Web API and ECMAScript 6 features which let our web UI components library
```
Comment (fshovchko) [inline README.md:34]:
```suggestion
ESL is based just on Web API and ECMAScript 6 features which let our web UI components library 
```

File: README.md [hunk 2]
```diff
@@ -9,22 +9,46 @@
...
+While looking for the best UI component library, keep in mind that it should be compatible with other libraries 
+and frameworks. And ESL is!
+
+The UI component library allows you to access the toolkit full of reusable components. 
+You can utilize these web components to complement projects or build your infrastructure from the ground 
+up using ESL alone. Our web UI components library is accurate and you can easily work around styles 
+to make your projects consistent.
+
+ESL based just on Web API and ECMAScript 6 features which let our web UI components library 
+work well across all modern browsers such as Firefox, Chrome, Safari, Opera, or Edge. ESL can
```
Comment (fshovchko) [inline README.md:35]:
```suggestion
to work well across all modern browsers such as Firefox, Chrome, Safari, Opera, or Edge. ESL can 
```

File: README.md [hunk 3]
```diff
@@ -9,22 +9,46 @@
...
+<p align="center" >★ <b>Check out our UI component library and support it with a star</b> ★</p>
+
+## Overview
+
+Exadel Smart Library, or ESL for short, is a free web UI components library that reduces routine tasks 
+and lets you efficiently create websites with multiple web components at hand. 
+While looking for the best UI component library, keep in mind that it should be compatible with other libraries 
+and frameworks. And ESL is!
+
+The UI component library allows you to access the toolkit full of reusable components.
```
Comment (fshovchko) [inline README.md:29]:
```suggestion
The UI component library allows you to access the toolkit with plenty of reusable components. 
```

File: README.md [hunk 4]
```diff
@@ -9,22 +9,46 @@
...
+to make your projects consistent.
+
+ESL based just on Web API and ECMAScript 6 features which let our web UI components library 
+work well across all modern browsers such as Firefox, Chrome, Safari, Opera, or Edge. ESL can 
+also work on IE11 when the whole list of polyfills is provided (see [Browser support & Polyfills](https://github.com/exadel-inc/esl/blob/HEAD/docs/BROWSER_SUPPORT.md) for more details). 
+
 
 ## Library Structure
+### Base Components
+With our UI component library, you get flexible syntax sugar meant for building over basic components and controlling the life cycle of those components.
```
Comment (fshovchko) [inline README.md:41]:
```suggestion
With our UI component library, you are getting flexible syntax sugar meant for building over basic components and controlling the life cycle of those components.
```

**Comment by ala-n** [comment]
Waits review from the Content Team

File: README.md (hunk 1 above)
Comment (ala-n) [reply README.md]: Updated

File: README.md (hunk 3 above)
Comment (ala-n) [reply README.md]: Updated

File: README.md [hunk 5]
```diff
@@ -9,22 +9,56 @@
...
+- [Installation Guide](#installation-guide)
+- [Roadmap](#roadmap)
+- [Development Information for Contributors](#development-information-for-contributors)
+
+## Overview
+
+Exadel Smart Library, or ESL for short, is a free web UI components library that reduces routine tasks and lets 
+you efficiently create websites with multiple web components at hand. 
+While looking for the best UI component library, keep in mind that it should be compatible with other libraries 
+and frameworks. And ESL is!
```
Comment (fshovchko) [inline README.md:34]:
```suggestion
and frameworks. And ESL does!
```

File: README.md [hunk 6]
```diff
@@ -9,22 +9,56 @@
 [![build](https://img.shields.io/github/workflow/status/exadel-inc/esl/validate/main?style=for-the-badge)](https://github.com/exadel-inc/esl/actions/workflows/validate.yml)
 [![License](https://img.shields.io/badge/license-MIT-green.svg?style=for-the-badge)](./README.md)
 
-Exadel Smart Library (**ESL**) is a web components based library that gives you a set of **lightweight** and **flexible** custom elements to easily create basic UX modules and make your websites super fast.
+Exadel Smart Library (**ESL**) is an open-source **lightweight** and **flexible** UI component library based on web components.
+Build UX modules with custom components provided by our library and make sites work faster.
```
Comment (fshovchko) [inline README.md:13]:
```suggestion
Build UX modules with custom components provided by our library to make sites work faster. 
```

**Review by julia-murashko — APPROVED** [review]

File: README.md [hunk 7]
```diff
@@ -9,22 +9,56 @@
 [![build](https://img.shields.io/github/workflow/status/exadel-inc/esl/validate/main?style=for-the-badge)](https://github.com/exadel-inc/esl/actions/workflows/validate.yml)
 [![License](https://img.shields.io/badge/license-MIT-green.svg?style=for-the-badge)](./README.md)
 
-Exadel Smart Library (**ESL**) is a web components based library that gives you a set of **lightweight** and **flexible** custom elements to easily create basic UX modules and make your websites super fast.
+Exadel Smart Library (**ESL**) is an open-source **lightweight** and **flexible** UI component library based on web components.
```
Comment (NastaLeo) [inline README.md:12]: I suggest write "UI components (plural form) library" throughout the document.

**Review by NastaLeo — APPROVED** [review]

File: README.md [hunk 8]
```diff
@@ -33,12 +67,13 @@ Exadel Smart Library (**ESL**) is a web components based library that gives you
 - ##### [ESL Animate](https://github.com/exadel-inc/esl/blob/HEAD/src/modules/esl-animate/README.md) (beta)
 - ##### [ESL Popup](https://github.com/exadel-inc/esl/blob/HEAD/src/modules/esl-popup/README.md) (beta)
 
-### Form Components
 - ##### [ESL Select](https://github.com/exadel-inc/esl/blob/HEAD/src/modules/esl-forms/esl-select/README.md)
 - ##### [ESL Select List](https://github.com/exadel-inc/esl/blob/HEAD/src/modules/esl-forms/esl-select-list/README.md)
 
 ### Utilities
-- ##### [ESL Base Element](https://github.com/exadel-inc/esl/blob/HEAD/src/modules/esl-base-element/README.md)
+Our web UI components library uses utilities to build components with simpler syntax sugar and a big set of helpers.
```
Comment (ala-n) [inline README.md:74]:
```suggestion
Our web UI components library uses utilities to build components using simplifying syntax sugar and a big set of helpers.
```

File: README.md (hunk 5 above)
Comment (ala-n) [reply README.md:34]: It looks like the original one is more proper here.

**Review by fshovchko — APPROVED** [review]

**Review by dshovchko — APPROVED** [review]

**Review by abarmina — APPROVED** [review]

File: README.md (hunk 7 above)
Comment (ala-n) [reply README.md:12]: Have confirmation from the SEO team that's the form of keyword important here and unfortunately
"UI component library" is a popular keyword but the "UI components library" is not (

**Pull request merged**